### PR TITLE
Remove Push Notifications BETA

### PR DESCRIPTION
**Pull request by lukabratos**
### Description of the pull request

See title.

#### Why is the change necessary?

Please use Pusher Beams, our new Push Notifications product: www.pusher.com/beams
If you're planning to migrate, check our migration guide:
https://pusher.com/docs/push_notifications/migration

### CODE REVIEW

File: iOS Example Obj-C/iOS Example Obj-C/AppDelegate.m [hunk 1]
```diff
@@ -19,58 +18,7 @@ @implementation AppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     self.pusher = [[Pusher alloc] initWithKey:@"YOUR_APP_KEY"];
```
Comment (TomKemp) [inline iOS Example Obj-C/iOS Example Obj-C/AppDelegate.m]: I don't think the app delegate needs an instance of Pusher. It was only being used for push-notifications.

File: iOS Example Swift/iOS Example Swift/AppDelegate.swift [hunk 2]
```diff
@@ -8,68 +8,15 @@
...
 import UIKit
 import PusherSwift
-import UserNotifications
 
 @UIApplicationMain
-class AppDelegate: UIResponder, UIApplicationDelegate, PusherDelegate {
+class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
     let pusher = Pusher(key: "YOUR_APP_KEY")
```
Comment (TomKemp) [inline iOS Example Swift/iOS Example Swift/AppDelegate.swift]: Same here. I don't think the app delegate needs an instance of `Pusher`. It was only being used for push-notifications.

File: macOS Example Swift/macOS Example Swift/AppDelegate.swift [hunk 3]
```diff
@@ -10,55 +10,15 @@ import Cocoa
 import PusherSwift
 
 @NSApplicationMain
-class AppDelegate: NSObject, NSApplicationDelegate, PusherDelegate {
+class AppDelegate: NSObject, NSApplicationDelegate {
 
     let pusher = Pusher(key: "YOUR_APP_KEY")
```
Comment (TomKemp) [inline macOS Example Swift/macOS Example Swift/AppDelegate.swift]: Same again. I don't think the app delegate needs an instance of Pusher. It was only being used for push-notifications.

File: iOS Example Obj-C/iOS Example Obj-C/AppDelegate.m (hunk 1 above)
Comment (lukabratos) [reply iOS Example Obj-C/iOS Example Obj-C/AppDelegate.m]: You're correct. We just had example code related to push notification here.
I have intentionally left at least instance of Pusher client in here if we would potentially add more examples - maybe related to some other functionality that this SDK provides but we can do that later. I'll remove this code.

File: Sources/PusherSwift.swift [hunk 4]
```diff
@@ -11,16 +11,11 @@ let CLIENT_NAME = "pusher-websocket-swift"
...
         willSet {
             self.connection.delegate = newValue
-#if os(iOS) || os(OSX)
-            self.nativePusher.delegate = newValue
-#endif
         }
     }
     private let key: String
 
 #if os(iOS) || os(OSX)
```
[the remark this answers is not in the document]
Comment (lukabratos) [reply Sources/PusherSwift.swift]: Nice catch, thanks 👍

**Comment by TomKemp** [comment]
Just leaving a note so I remember to check further before merging this: I think the `TaskQueue` dependency is only used for the old native push notifications, so that dependency can be removed.

**Pull request merged**